docs/source/developers/python.rst

```diff
@@ -106,8 +109,8 @@ Building on Linux and MacOS
 System Requirements
 -------------------
 
-On macOS, any modern XCode (6.4 or higher; the current version is 10) is
-sufficient.
+On macOS, any modern XCode (6.4 or higher; the current version is 13) is
+sufficient. All you need is to have XCode installed, no additional configuration is needed.
```

Comment:
Is this useful? The preceding sentence already says that "XCode is sufficient".

Reply:
Actually (I am a new Mac user so I am still learning) the easiest way would be to only install Xcode Command Line Tools via `xcode-select --install` or via Homebrew. Installing full Xcode is taking way to long, it stopped twice in my case and at the end I still wasn't sure if it worked.

Maybe it's just me but additional sentence can help ...

Reply:
Well, in this case the sentence can be about the Xcode Command Line Tools.